index.js

```diff
@@ -21,6 +21,9 @@ var KILL_SIGNAL = 'SIGTERM';
 var reloading = false;
 var reloadPedding = false;
 function reload(count) {
+  if (cluster.isWorker) {
```

Comment:
the reload action is executing on master process, cluster fork should be exists.